.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Thanks for contributing! -->
+## Summary
+<!-- What conceptually is this PR introducing? If context is already provided from the JIRA ticket, still place it in the
+Pull Request as you should not make the reviewer do digging for a basic summary. -->
+
+## Changes in this PR
+<!-- What changes did you make to the code? What new APIs (public or private) were added, removed, or edited to generate
+the desired outcome explained in the above summary? -->
+
+## Test Plan
```

Comment:
Considering changing to "Testing". Plan feels forward looking, while the description below is backward looking.

Reply:
Changed!